### PR TITLE
FIX: load_nwb creates integer EventData

### PR DESCRIPTION
**Pull request by KatharineShapcott**
Changes Summary
----------------
- Use int datatype for loaded EventData

Reviewer Checklist
------------------
- [ ] Are testing routines present?
- [ ] Do objects in the global package namespace perform proper parsing of their input? 
- [ ] Are all docstrings complete and accurate?
- [ ] Is the CHANGELOG.md up to date?

### CODE REVIEW

File: syncopy/io/load_nwb.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def load_nwb(filename, memuse=3000, container=None):
 
         evtData = EventData(dimord=["sample","eventid","chans"], filename=filename)
         h5evt = h5py.File(evtData.filename, mode="w")
-        evtDset = h5evt.create_dataset("data", dtype=np.result_type(*ttlDtypes),
+        evtDset = h5evt.create_dataset("data", dtype=int,
```
Comment (tensionhead) [inline syncopy/io/load_nwb.py:200]: Okay that looks much simpler of course.. we can be sure only integers are to be expected here?

Comment (KatharineShapcott) [reply syncopy/io/load_nwb.py:200]: No they are floats but they need to be converted otherwise it crashes.

Comment (tensionhead) [reply syncopy/io/load_nwb.py:200]: Ok.. so does that mean there could be events encoded with floats? I think we only enforce integere types for `SpikeData` now, not for `EventData`?!

Comment (KatharineShapcott) [reply syncopy/io/load_nwb.py:200]: Events should always be integers but they're not stored like that in NWB. No it's enforced for both, this crashes now.

Comment (tensionhead) [reply syncopy/io/load_nwb.py:200]: Uh, but apparently rightfully so :slightly_smiling_face: Ok then let's do it like you proposed!

Comment (KatharineShapcott) [reply syncopy/io/load_nwb.py:200]: We could print a warning that we're casting?

Comment (tensionhead) [reply syncopy/io/load_nwb.py:200]: Maybe at the debug level, we have proper logging now with #409 But as you said, the nwb reader should anyways be considered experimental :sweat_smile:

**Pull request merged**